docs/website/src/site.less

```
@@ -1,28 +1,28 @@
-@import "./common/reboot.less";
-@import "./common/typography.less";
-@import "./common/colors.less";
-@import "./common/markdown.less";
+@import './common/reboot.less';
+@import './common/typography.less';
+@import './common/colors.less';
+@import './common/markdown.less';
 
-@import "./common/btns.less";
-@import "./common/containers.less";
-@import "./common/helpers.less";
-@import "./common/scrollbar.less";
-@import "./eak-zoom-image/eak-zoom-image.less";
+@import './common/btns.less';
+@import './common/containers.less';
+@import './common/helpers.less';
+@import './common/scrollbar.less';
+@import './eak-zoom-image/eak-zoom-image.less';
 
-@import "./page/sidebar-layout.less";
-@import "./page/landing-layout.less";
-@import "./page/articles-layout.less";
+@import './page/sidebar-layout.less';
+@import './page/landing-layout.less';
+@import './page/articles-layout.less';
 
-@import "./navigation/navigation.less";
-@import "./landing/landing.less";
+@import './navigation/navigation.less';
+@import './landing/landing.less';
 
-@import "../node_modules/@exadel/esl/modules/esl-utils/all";
-@import "../node_modules/@exadel/esl/modules/esl-image/core";
-@import "../node_modules/@exadel/esl/modules/esl-scrollbar/core";
-@import "../node_modules/@exadel/esl/modules/esl-trigger/core";
-@import "../node_modules/@exadel/esl/modules/esl-toggleable/core";
-@import "../node_modules/@exadel/esl/modules/esl-panel/core";
-@import "../node_modules/@exadel/esl/modules/esl-panel-group/core";
-@import "../node_modules/@exadel/esl/modules/esl-tab/core";
+@import '../node_modules/@exadel/esl/modules/esl-utils/all';
+@import '../node_modules/@exadel/esl/modules/esl-image/core';
+@import '../node_modules/@exadel/esl/modules/esl-scrollbar/core';
+@import '../node_modules/@exadel/esl/modules/esl-trigger/core';
+@import '../node_modules/@exadel/esl/modules/esl-toggleable/core';
+@import '../node_modules/@exadel/esl/modules/esl-panel/core';
+@import '../node_modules/@exadel/esl/modules/esl-panel-group/core';
+@import '../node_modules/@exadel/esl/modules/esl-tab/core';
```

I think this should work
```suggestion
@import '@exadel/esl/modules/esl-utils/all';
@import '@exadel/esl/modules/esl-image/core';
@import '@exadel/esl/modules/esl-scrollbar/core';
@import '@exadel/esl/modules/esl-trigger/core';
@import '@exadel/esl/modules/esl-toggleable/core';
@import '@exadel/esl/modules/esl-panel/core';
@import '@exadel/esl/modules/esl-panel-group/core';
@import '@exadel/esl/modules/esl-tab/core';
```